openapi.yaml

```
@@ -2418,15 +2420,15 @@ paths:
             application/json:
               schema:
                 properties:
-                  '': { type: string, example: 'Backup configuration and all executions deleted.' }
+                  message: { type: string, example: 'Backup configuration and all executions deleted.' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)